.github/workflows/codeql.yaml

```
@@ -4,21 +4,103 @@ name: codeql
 on:
   push:
     branches: [main]
+    tags:
+    - '*'
   pull_request:
     branches: [main]
   schedule:
     - cron: '0 8 * * 1' # Monday - 8am - UTC
+  workflow_dispatch:
 
 jobs:
+  build:
+    permissions:
+      contents: read
+      packages: write
+    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
+    with:
+      name: runner-manager
+      namespace: scality
+
+  trivy:
+    needs: build
+    name: Trivy Vulnerability Scan for Code
+    uses: scality/workflows/.github/workflows/trivy.yaml@v1
+    with:
+      name: runner-manager
+      namespace: scality
+
+  trivy-sbom:
+    needs: build
+    name: Trivy generate SBOM
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy in GitHub SBOM mode and submit results to Dependency Snapshots
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
```

scan the image not the fs
```suggestion
          scan-type: 'image'
```